### PR TITLE
PAR-620: Fixed issue employee number not showing (none).

### PR DESCRIPTION
**Pull request by stephenb01**

### CODE REVIEW

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsOrganisationDetailsForm.php [hunk 1]
```diff
@@ -173,7 +173,7 @@ public function buildForm(array $form, FormStateInterface $form_state, ParDataPa
       '#collapsed' => FALSE,
     ];
 
-    if ($par_data_organisation->get('employees_band')->getString() !== '0') {
+    if ($par_data_organisation->get('employees_band')->getString() !== '') {
```
Comment (jon-edery) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsOrganisationDetailsForm.php]: FALSE would be a more robust option here.

Comment (stephenb01) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsOrganisationDetailsForm.php]: It needs to be '' otherwise it doesn't work.

Comment (kalpaitch) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsOrganisationDetailsForm.php]: Maybe `!$par_data_organisation->get('employees_band')->isEmpty()`

Comment (jon-edery) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsOrganisationDetailsForm.php]: empty string should evaluate to FALSE:  

 When converting to boolean, the following values are considered FALSE as well as the following:

    the boolean FALSE itself
    the integer 0 (zero)
    the float 0.0 (zero)
    the empty string, and the string "0"
    an array with zero elements
    the special type NULL (including unset variables)
    SimpleXML objects created from empty tags

**Pull request merged**